src/ui/Buttons/Button.jsx

```diff
@@ -15,7 +15,7 @@ export const Button = styled.button`
   font-size: 14px;
   font-weight: 500;
   background-color: var(--fill, var(--ds-background-brand-bold));
-  color: var(--text, var(--ds-text-inverse));
+  color: var(--text), var(--ds-text-inverse));
```

Comment:
`color: var(--text), var(--ds-text-inverse));`

you have here an error. check it

Reply:
yea, I've added here parentheses, it was missing, i don't know why..